pkgs/development/libraries/glibc/0001-Fix-build-on-case-insensative-filesystems.patch

```diff
@@ -0,0 +1,1542 @@
+ 	$(make-target-directory)
+ 	rm -f $@
+ 	ln $< $@
+@@ -161,7 +161,7 @@ do-tests-clean common-mostlyclean: tst-x86_64-1-clean
+ tst-x86_64-1-clean:
+ 	-rm -rf $(objpfx)x86_64
+ 
+-$(objpfx)x86_64/tst-platformmod-2.os: $(objpfx)tst-platformmod-2.os
++$(objpfx)x86_64/tst-platformmod-2.o_shared: $(objpfx)tst-platformmod-2.o_shared
+ 	$(make-target-directory)
+ 	rm -f $@
+ 	ln $< $@
+diff --git a/sysdeps/x86_64/fpu/Makefile b/sysdeps/x86_64/fpu/Makefile
+index 2b7d69bb50..5fa29f61fc 100644
+--- a/sysdeps/x86_64/fpu/Makefile
++++ b/sysdeps/x86_64/fpu/Makefile
+@@ -80,15 +80,15 @@ $(objpfx)test-double-libmvec-alias-avx2-mod.so: \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-alias-main: \
+-  $(objpfx)test-double-libmvec-alias-mod.os \
++  $(objpfx)test-double-libmvec-alias-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-alias-avx-main: \
+-  $(objpfx)test-double-libmvec-alias-avx-mod.os \
++  $(objpfx)test-double-libmvec-alias-avx-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-alias-avx2-main: \
+-  $(objpfx)test-double-libmvec-alias-avx2-mod.os \
++  $(objpfx)test-double-libmvec-alias-avx2-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias: \
+@@ -107,15 +107,15 @@ $(objpfx)test-float-libmvec-alias-avx2-mod.so: \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias-main: \
+-  $(objpfx)test-float-libmvec-alias-mod.os \
++  $(objpfx)test-float-libmvec-alias-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias-avx-main: \
+-  $(objpfx)test-float-libmvec-alias-avx-mod.os \
++  $(objpfx)test-float-libmvec-alias-avx-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias-avx2-main: \
+-  $(objpfx)test-float-libmvec-alias-avx2-mod.os \
++  $(objpfx)test-float-libmvec-alias-avx2-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-sincos: \
+@@ -165,7 +165,7 @@ $(objpfx)test-double-libmvec-alias-avx512-mod.so: \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-alias-avx512-main: \
+-  $(objpfx)test-double-libmvec-alias-avx512-mod.os \
++  $(objpfx)test-double-libmvec-alias-avx512-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias-avx512: \
+@@ -174,7 +174,7 @@ $(objpfx)test-float-libmvec-alias-avx512-mod.so: \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-float-libmvec-alias-avx512-main: \
+-  $(objpfx)test-float-libmvec-alias-avx512-mod.os \
++  $(objpfx)test-float-libmvec-alias-avx512-mod.o_shared \
+   $(objpfx)../mathvec/libmvec_nonshared.a $(libmvec)
+ 
+ $(objpfx)test-double-libmvec-sincos-avx512: \
+-- 
+2.18.0
+
```

Comment:
This diff is really big, could we fetch this from somewhere?

Reply:
Yeah I am upstreaming that now before merging.